README.md

```diff
@@ -1,2 +1,50 @@
 # 3D-Vac
 Repository of the eScience Center and Radboudumc "Personalized cancer vaccine design through 3D modelling boosted geometric learning" collaboartive project (OEC 2021).
+
+## Repository structure
+
+I used [Cookiecutter Data Science](https://drivendata.github.io/cookiecutter-data-science/#starting-a-new-project) to create this project structure. The following is the directory structure:
+
+```
+├── LICENSE
+├── README.md          <- The top-level README for developers using this project.
+├── data
+│   ├── external       <- Data from third party sources.
+│   ├── interim        <- Intermediate data that has been transformed.
+│   ├── processed      <- The final, canonical data sets for modeling.
+│   └── raw            <- The original, immutable data dump.
+│
+├── docs               <- For useful documentation
+│
+├── models             <- Trained and serialized models, model predictions, or model summaries
+│
+├── notebooks          <- Jupyter notebooks. Naming convention is a number (for ordering),
```

Comment:
Do we want to keep notebooks separate from the scripts? 
I would vote for keeping only scripts.

Reply:
It's usually useful to have interactive notebooks during the development, even if absolutely not necessary. One reason to keep them anyway (not all of us use notebooks, but some of us do) is that they can become tutorials for later user usage, so we'll probably need them at a certain point. So I would keep the folder, and If not all of us are going to use it for the development part, is perfectly fine.